thirdparty/build-thirdparty.sh

```diff
@@ -685,6 +685,18 @@ build_cctz() {
     make -j$PARALLEL && make install
 }
 
+#mustache
+build_mustache() {
```

Comment:
In `https://github.com/henryr/cpp-mustache/` it said:
`Easiest way is to drop mustache.{cc,h} in your project.`
so why not just introduce the 2 files? So that we don't need to update our compile env.